app/controllers/api/v1/comments_controller.rb

```diff
@@ -3,22 +3,22 @@
 module Api
   module V1
     class CommentsController < ApplicationController
-      before_action :authenticate_request, only: %i[index create update]
-      before_action :set_comment, only: %i[update]
+      before_action :authenticate_request, only: %i[index create update destroy]
+      before_action :set_comment, only: %i[update destroy]
+      before_action :authorize_user, only: %i[create update destroy]
 
       def index
         @comment = Comment.order('created_at DESC')
```

Comment:
Is a naming convention if the variable can contain more than 1 result, to user plural names:
```suggestion
        @comments = Comment.order('created_at DESC')
```